src/Symfony/Bundle/FrameworkBundle/EventListener/SuggestMissingPackageSubscriber.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\EventListener;
+
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleErrorEvent;
+use Symfony\Component\Console\Exception\CommandNotFoundException;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Suggests a package, that should be installed (via composer),
+ * if the package is missing, and the input command namespace can be mapped to a Symfony bundle.
+ *
+ * @author Przemysław Bogusz <przemyslaw.bogusz@tubotax.pl>
+ *
+ * @internal
+ */
+final class SuggestMissingPackageSubscriber implements EventSubscriberInterface
+{
+    private const PACKAGES = [
+        'doctrine' => [
+            'fixtures' => ['DoctrineFixturesBundle', 'doctrine/doctrine-fixtures-bundle --dev'],
+            'mongodb' => ['DoctrineMongoDBBundle', 'doctrine/mongodb-odm-bundle'],
+            '_default' => ['Doctrine ORM', 'symfony/orm-pack'],
+        ],
+        'generate' => [
+            '_default' => ['SensioGeneratorBundle', 'sensio/generator-bundle'],
+        ],
+        'make' => [
+            '_default' => ['MakerBundle', 'symfony/maker-bundle --dev'],
+        ],
+        'server' => [
+            'dump' => ['VarDumper Component', 'symfony/var-dumper --dev'],
+            '_default' => ['WebServerBundle', 'symfony/web-server-bundle --dev'],
+        ],
+    ];
+
+    public function onConsoleError(ConsoleErrorEvent $event): void
+    {
+        if (!$event->getError() instanceof CommandNotFoundException || empty($input = $event->getInput()->getFirstArgument())) {
```

Comment:
Can you remove the usage of `empty`. We tend to avoid using it as many "valid" strings are "empty" like `0` for instance.

Reply:
Done